deployment/environments/hosts.local

```diff
@@ -165,16 +165,15 @@ hosts_vitamui_mongod
 
 [hosts_vitamui_mongod]
 # EDIT: Mandatory
-# WARNING: put only one server for this service, not more !
 # Mandatory params
-#  - mongo_rs_bootstrap=true (default: false)
+#  - mongo_rs_bootstrap=true (default: false) ; mandatory for 1 node of the shard, some init commands will be executed on it
```

Comment:
"mandatory for 1 node" peut être interprété comme il faut au moins sur 1 noeud

Reply:
J'ai repris le même commentaire que sur Vitam, à voir si on le modifie mais dans ce cas là faut le modifier partout :)